tests/ui/perfherder/alerts-view/status_dropdown_test.jsx

```diff
@@ -85,3 +89,231 @@ test("Tags modal opens from 'Edit tags'", async () => {
 
   expect(modal).toBeInTheDocument();
 });
+
+test('Triage to countdown shows 3 working days left', async () => {
+  const alert = testAlertSummaries[0];
+  alert.created = '2022-02-07T11:41:31.419156';
```

Comment:
To make the tests easier to follow you can put here a comment saying that you are setting the created date to Monday. I would do this to all the tests.